src/NzbDrone.Core/DecisionEngine/QualityUpgradableSpecification.cs

```diff
@@ -1,74 +0,0 @@
-using NLog;
-using NzbDrone.Core.Profiles;
-using NzbDrone.Core.Qualities;
-
-namespace NzbDrone.Core.DecisionEngine
-{
-    public interface IQualityUpgradableSpecification
```

Comment:
Modified the name to IUpgradableSpecification since language is also involved in the method now.